incubator/kured/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
should this be conditional on rbac.create too?

Reply:
Agreed and fixed.